components/ingest-service/integration_test/suite_test.go

```diff
@@ -359,7 +358,27 @@ func createServices(s *Suite) {
 	// // To test the 'mark missing nodes for deletion' job
 	// res, err := suite.JobSchedulerServer.MarkMissingNodesForDeletion(ctx, &req)
 	// ```
-	s.JobSchedulerServer = server.NewJobSchedulerServer(s.ingest, s.JobScheduler, s.ConfigManager)
+	jobManager, err := cereal.NewManager(postgres.NewPostgresBackend(postgresUrl))
+	if err != nil {
+		logrus.WithError(err).Fatal("could not create job manager")
+	}
+
```

Comment:
⚠️ `defer jobManager.Stop()`!!

Reply:
That can be added to `GlobalTeardown` on line 106.